main.go

```diff
@@ -60,17 +61,19 @@ func main() {
 	usage := fmt.Sprintf(`%s.
 
   Usage:
-  %s [--no-publish --sources=<sources>]
+  %s [--no-publish --sources=<sources> --label-whitelist=<pattern>]
   %s -h | --help
   %s --version
 
   Options:
-  -h --help           Show this screen.
-  --version           Output version and exit.
-  --sources=<sources> Comma separated list of feature sources.
-                      [Default: cpuid,rdt,pstate]
-  --no-publish        Do not publish discovered features to the cluster-local
-                      Kubernetes API server.`,
+  -h --help                   Show this screen.
+  --version                   Output version and exit.
+  --sources=<sources>         Comma separated list of feature sources.
+                              [Default: cpuid,rdt,pstate]
+  --no-publish                Do not publish discovered features to the cluster-local
+                              Kubernetes API server.
+  --label-whitelist=<pattern> Regex to apply to labels before adding.
```

Comment:
How about "Regular expression to filter label names to publish to the Kubernetes API server."?